vispy/visuals/graphs/layouts/__init__.py

```diff
@@ -0,0 +1,52 @@
+import inspect
+
+from .random import random
+from .circular import circular
+from .force_directed import fruchterman_reingold
+
+
+_layout_map = {
+    'random': random,
+    'circular': circular,
+    'force_directed': fruchterman_reingold,
+    'spring_layout': fruchterman_reingold
+}
+
+AVAILABLE_LAYOUTS = _layout_map.keys()
```

Comment:
on Py3k `.keys()` is a generator, so you need to do `list(...)` or `tuple(...)`, and we want it to be immutable so use the latter